server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/MIMEMessageConverter.java

```diff
@@ -104,6 +106,7 @@ public MIMEMessageConverter() {
             messageBuilder.setBody(createMultipart(creationMessageEntry.getValue(), messageAttachments));
         } else {
             messageBuilder.setBody(createTextBody(creationMessageEntry.getValue()));
+            messageBuilder.setContentTransferEncoding(QUOTED_PRINTABLE);
```

Comment:
This is the job of the writer, here is the methods chain:
DefaultMessageWriter.writeMessage.writeEntity.encodeStream